api/nakadi-event-bus-api.yaml

```diff
@@ -0,0 +1,1232 @@
+            items:
+              $ref: '#/definitions/Event'
+        - name: Content-Encoding
+          in: header
+          required: false
+          description: |
+            Allow the publisher to gzip compress the payload by setting this header to "gzip".
+            For Example "Content-Encoding: gzip"
+          type: string
+        - name: X-Flow-Id
+          in: header
+          description: |
+            The flow id of the request, which is written into the logs and passed to called services. Helpful
+            for operational troubleshooting and log analysis.
+          type: string
+          format: flow-id
+      responses:
+        '201':
+          description: All events are submitted successfully.
+        '401':
+          description: Not authenticated
+          schema:
+            $ref: '#/definitions/Problem'
+        '403':
+          description: Not allowed
+          schema:
+            $ref: '#/definitions/Problem'
+        '404':
+          description: |
+            Topic not found.
+          schema:
+            $ref: '#/definitions/Problem'
+        '405':
+          description: |
+            Method not allowed. This topic does not support direct POSTing of Events. Use equivalent POST
+            /event-types/{name}/events instead.
+          schema:
+            $ref: '#/definitions/Problem'
+        '422':
+          description: Unprocessable Entity
+          schema:
+            $ref: '#/definitions/Problem'
+        '500':
+          description: Server error
+          schema:
+            $ref: '#/definitions/Problem'
+
+        '503':
+          description: Not available
+          schema:
+            $ref: '#/definitions/Problem'
+
+  '/topics/{topic}/partitions':
+    get:
+      tags:
+        - topic-api
+        - management-api
+      description: Lists the `Partition`s for the given topic
+      parameters:
+        - name: topic
+          in: path
+          description: Topic name
+          type: string
+          required: true
+        - name: X-Flow-Id
+          in: header
+          description: |
+            The flow id of the request, which is written into the logs and passed to called services. Helpful
+            for operational troubleshooting and log analysis.
+          type: string
+          format: flow-id
+      responses:
+        '200':
+          description: OK
+          schema:
+            type: array
+            description: An array of `Partition`s
+            items:
+              $ref: '#/definitions/Partition'
+        '404':
+          description: |
+            Topic not found.
+          schema:
+            $ref: '#/definitions/Problem'
+        '500':
+          description: Server error
+          schema:
+            $ref: '#/definitions/Problem'
+
+  '/topics/{topic}/partitions/{partition}':
+    get:
+      tags:
+        - topic-api
+        - management-api
+      description: Returns the given `Partition` of this topic
```

Comment:
editorial: Returns the given Partition (including its management cursors) of this topic.